### PR TITLE
Update Bob's Books readme with instructions on how to test the application

### PR DESCRIPTION
**Pull request by mgianatagh**
Added a section to the README of the Bob's Books sample application that describes how to test the application after it is installed.  This change is being done in response to one of our users asking how to test Bob's books.

### CODE REVIEW

File: examples/bobs-books/README.md [hunk 1]
```diff
@@ -108,3 +108,43 @@ cluster if you are using a multi-cluster environment.
    ```
    kubectl apply -f bobs-books-binding.yaml
    ```
+
+## Access the sample application endpoints
+
+1. Get the external address of the `Istio` ingress gateway
```
Comment (markxnelson) [inline examples/bobs-books/README.md:114]: these Istio's don't need backticks

File: examples/bobs-books/README.md [hunk 2]
```diff
@@ -108,3 +108,43 @@ cluster if you are using a multi-cluster environment.
...
+
+    This table shows the application endpoints using the following example values:
+    * Environment Name: `demo`
+    * External IP address is `123.456.78.901`
+    * DNS Zone: `verrazzano.demo.com`
+    
+    | Description| End Point | Credentials |
+    | --- | --- | --- |
+    | Bobby's Books | http://123.456.78.901/bobbys-front-end | |
+    | Bobby's Books WebLogic Console | http://123.456.78.901/console | weblogic/welcome1 |
```
Comment (markxnelson) [inline examples/bobs-books/README.md:145]: the password would be whatever they chose when they installed it, not necessarily welcome1 - we should point that out, and remind them to use secure passwords

File: examples/bobs-books/README.md (hunk 1 above)
Comment (rosemarymarano) [inline examples/bobs-books/README.md:114]: gateway -> gateway. (period at the end of each sentence; globally)

File: examples/bobs-books/README.md [hunk 3]
```diff
@@ -108,3 +108,43 @@ cluster if you are using a multi-cluster environment.
...
    kubectl apply -f bobs-books-binding.yaml
    ```
+
+## Access the sample application endpoints
+
+1. Get the external address of the `Istio` ingress gateway
+
+    Access to the sample application is through the ingress gateway of the `Istio` mesh.
+
+    Execute this command to get the external IP address of the `Istio` ingress gateway:
```
Comment (rosemarymarano) [inline examples/bobs-books/README.md:118]: Execute -> Run (execute (v) Do not use when you mean performing a programming instruction. Use run or process instead. Globally.)

File: examples/bobs-books/README.md [hunk 4]
```diff
@@ -108,3 +108,43 @@ cluster if you are using a multi-cluster environment.
...
+
+1. Get the password for telemetry endpoints
+
+    Execute this command to get the password that was generated for the telemetry components:
+    ```
+    kubectl get secret --namespace verrazzano-system verrazzano -o jsonpath={.data.password} | base64 --decode; echo
+    ```
+1. Test the sample application endpoints
+
+    This table shows the application endpoints using the following example values:
```
Comment (rosemarymarano) [inline examples/bobs-books/README.md:137]: This table -> The following table
using the following example values: -> using these example values:

File: examples/bobs-books/README.md [hunk 5]
```diff
@@ -108,3 +108,43 @@ cluster if you are using a multi-cluster environment.
...
+    * Environment Name: `demo`
+    * External IP address is `123.456.78.901`
```
Comment (rosemarymarano) [inline examples/bobs-books/README.md:139]: address is -> address: (to be consistent with the others)

File: examples/bobs-books/README.md [hunk 6]
```diff
@@ -108,3 +108,43 @@ cluster if you are using a multi-cluster environment.
...
+    * DNS Zone: `verrazzano.demo.com`
+    
+    | Description| End Point | Credentials |
+    | --- | --- | --- |
+    | Bobby's Books | http://123.456.78.901/bobbys-front-end | |
+    | Bobby's Books WebLogic Console | http://123.456.78.901/console | weblogic/welcome1 |
+    | Robert's Books | http://123.456.78.901 | |
+    | ElasticSearch | https://elasticsearch.vmi.bobs-books-binding.demo.verrazzano.demo.com | verrazzano/<password> |
```
Comment (rosemarymarano) [inline examples/bobs-books/README.md:147]: ElasticSearch -> Elasticsearch
password in angle brackets -> does not show up, https://github.com/verrazzano/verrazzano/blob/66b7c0195ff5c17f73acfc3fc30a9d70be1ec009/examples/bobs-books/README.md

File: examples/bobs-books/README.md (hunk 2 above)
Comment (rosemarymarano) [inline examples/bobs-books/README.md:145]: weblogic/welcome1 -> `weblogic`/`welcome1` ( all the credentials should be in code font (surrounded by back ticks))

**Review by rosemarymarano — COMMENTED** [review]
Minor edits

**Review by rosemarymarano — APPROVED** [review]

**Review by markxnelson — APPROVED** [review]

**Pull request merged**